lib/components/OrderDialog/InitialOrder.tsx

```diff
@@ -62,10 +65,28 @@ export const InitialOrderScreen = ({
         ).cost
       : 0
 
-  // Create order quote when component mounts
```

Comment:
Why remove the useEffect? It will not create the order when the user is already signed in